af/analysis/analysis.py

```diff
@@ -0,0 +1,879 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""ooni-pipeline: * -> Analysis
+
+Creates and updates unlogged tables.
+Shows confirmed correlated by country, ASN, input URL over time.
+
+Inputs: Database tables:
+                      confirmed_stats
+                      msm_count_by_day_by_country
+                      countries
+                      interesting_inputs
+
+Outputs: Dedicated unlogged database tables and charts
+          tables:
+              currently_blocked
+
+"""
+__version__ = "0.0.1"
+
+# Compatible with Python3.7 - linted with Black
+
+# TODO:
+# Overall datapoints count per country per day
+# Add ASN to confirmed_stats and use one table only if performance is
+# acceptable.
+# Move slicing and manipulation entirely in Pandas and drop complex SQL queries
+# Support feeder.py for continuous ingestion
+# Implement a crude precision metric based on msm_count and time window
+
+from argparse import ArgumentParser
+import datetime
+import os
+
+from sqlalchemy import create_engine
+import pandas as pd
+import psycopg2
+import matplotlib
+
+matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from metrics import setup_metrics
```

Comment:
I suggest you use the relative import syntax here to avoid confusing it with the module `metrics`.

That is doing this:
```
from .metrics import setup_metrics
```